### PR TITLE
get_decayconstant

### PR DESCRIPTION
**Pull request by EnricaBelfiore**

### CODE REVIEW

File: sandy/decay.py [hunk 1]
```diff
@@ -114,6 +114,40 @@ def get_halflives(self):
...
+                   DLAMBDA      LAMBDA
+           922350 2.21715e-20 3.12085e-17
+        """
+
+        decay_constant = {zam: {
+             "LAMBDA": dic['decay_constant'],
+             "DLAMBDA": dic['decay_constant_uncertainty'],
+             } for zam, dic in self.data.items()}
+
+        df = pd.DataFrame(decay_constant)
```
Comment (luca-fiorito-11) [inline sandy/decay.py:145]: I would do
```
df = pd.DataFrame(decay_constant).T
if with_uncertainty:
            return df
else:
            return df.LAMBDA
```
I am not sure `iat` is the best for this case

Comment (luca-fiorito-11) [reply sandy/decay.py:145]: But if you do this you need to say in the docstring that you get a series in output if with_uncertainty is false

File: sandy/decay.py [hunk 2]
```diff
@@ -333,6 +367,7 @@ def get_transition_matrix(self):
               .fillna(0)
         return T.reindex(T.columns.values, fill_value=0.0)
 
+
```
Comment (luca-fiorito-11) [inline sandy/decay.py:370]: PEP8, Remove this line

**Review by luca-fiorito-11 — CHANGES_REQUESTED** [review]

File: sandy/decay.py (hunk 1 above)
Comment (EnricaBelfiore) [reply sandy/decay.py]: Let me know if what I wrote is the correct way to explain the two output options

File: sandy/decay.py [hunk 3]
```diff
@@ -136,17 +137,16 @@ def get_decayconstant(self, with_uncertainty=True):
                    DLAMBDA      LAMBDA
            922350 2.21715e-20 3.12085e-17
```
Comment (luca-fiorito-11) [inline sandy/decay.py:138]: you should also test `with_uncertainty=False`

File: sandy/decay.py [hunk 4]
```diff
@@ -114,6 +114,40 @@ def get_halflives(self):
         series.index.name = "ZAM"
         return series
 
+    def get_decayconstant(self, with_uncertainty=True):
+        """
+        Extract decay constant and its uncertainty into dataframe.
+
+        Parameters
+        ----------
+        with_uncertainty : `bool`, optional, default is 'True'
```
Comment (luca-fiorito-11) [inline sandy/decay.py:123]: you did not describe this

**Comment by luca-fiorito-11** [comment]
@EnricaBelfiore, I merged the branch, although I'd like one more test

**Pull request merged**